tests/tensor/test_extra_ops.py

```diff
@@ -463,13 +463,12 @@ def test_axis(self):
 
         assert res.broadcastable == (False, True, False)
 
-    def test_invalid_axis(self):
-        # Test that trying to squeeze a non broadcastable dimension raises error
-        variable = TensorType(config.floatX, shape=(1, None))()
-        with pytest.raises(
-            ValueError, match="Cannot drop a non-broadcastable dimension"
-        ):
-            squeeze(variable, axis=1)
+    def test_non_broadcastable_axis(self):
+        # Test that we now allow squeezing of valid non-broadcastable dimension
```

Comment:
Taking the now will not age :)

```suggestion
        # Test that we allow squeezing of valid non-broadcastable dimension
```